src/bin/vip-wp.js

```diff
@@ -141,12 +141,18 @@ commandWrapper( {
 
 				const startsWithWp = line.startsWith( 'wp ' );
 				const empty = 0 === line.length;
+				const isShellCommand = 'wp shell' === line;
```

Comment:
I wonder if we should be more flexible in the check here, to handle any params (like `--url`) that get tacked on:
```suggestion
				const isShellCommand = line.startsWith( 'wp shell ' );
```